src/modules/configuration/tests/ConfigurationTests.cpp

```diff
@@ -0,0 +1,381 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <gtest/gtest.h>
```

Comment:
nit: Alphabetically sort headers

Reply:
That is not a rule and sometimes it could even break things (some headers require to be included in certain order).